### PR TITLE
Latvian: add multiple missing translations

### PR DESCRIPTION
**Pull request by MartinsLapsa**
Includes validation messages, storage units, and other.

### CODE REVIEW

File: rails/locale/lv.yml [hunk 1]
```diff
@@ -104,6 +107,10 @@ lv:
         zero: "%{count} mēneši"
         one: "%{count} mēnesis"
         other: "%{count} mēneši"
+      x_years:
```
Comment (pama) [inline rails/locale/lv.yml:110]: x_years key seems inexistent from the Action View locale:

https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionview/lib/action_view/locale/en.yml)

**Review by pama — CHANGES_REQUESTED** [review]

**Review by pama — CHANGES_REQUESTED** [review]

File: rails/locale/lv.yml [hunk 2]
```diff
@@ -104,6 +107,10 @@ lv:
...
+        zero: 0 gadi
+        one: 1 gads
```
Comment (digitalfrost) [inline rails/locale/lv.yml:112]:
```suggestion
```

File: rails/locale/lv.yml [hunk 3]
```diff
@@ -104,6 +107,10 @@ lv:
...
+        other: "%{count} gadi"
```
Comment (digitalfrost) [inline rails/locale/lv.yml:113]:
```suggestion
```

File: rails/locale/lv.yml (hunk 1 above)
Comment (digitalfrost) [inline rails/locale/lv.yml:110]:
```suggestion
```

**Comment by digitalfrost** [comment]
@pama -  this should fix the changes you requested.

**Comment by pama** [comment]
@digitalfrost thank you!

**Pull request merged**